rosa_architecture/rosa_policy_service_definition/rosa-sre-access.adoc

```diff
@@ -29,8 +29,8 @@ include::modules/how-service-accounts-assume-aws-iam-roles-in-sre-owned-projects
 [role="_additional-resources"]
 .Additional resources
 
-* For more information about the AWS IAM roles used by the cluster Operators, see https://docs.redhat.com/en/documentation/red_hat_openshift_service_on_aws/4/html/introduction_to_rosa/rosa-sts-about-iam-resources#rosa-sts-operator-roles_rosa-sts-about-iam-resources[Cluster-specific Operator IAM role reference].
+* link:https://docs.redhat.com/en/documentation/red_hat_openshift_service_on_aws/4/html/introduction_to_rosa/rosa-sts-about-iam-resources#rosa-sts-operator-roles_rosa-sts-about-iam-resources[Cluster-specific Operator IAM role reference].
```

Comment:
@Sangeeta can you approve the new styling update please, I cannot add words after content.
See the guidelines for `Additional resources` here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections
where you cannot add content after the brackets link.

Reply:
@fmcdonal LGTM